.github/workflows/ci.yaml

```diff
@@ -138,7 +138,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        repository: [cmd-registry-memory, cmd-registry-proxy-dns, sdk-kernel, sdk-vppagent, cmd-nsmgr]
+        repository:
+          - sdk-vppagent
```

Comment:
I think we can also add `cmd-nse-icmp-responder`

Reply:
Yep, done.